File: src/DrMITM.nim
import httpclient
import eop
import json
import os
intro()##Asks to enter URL##
var client = newHttpClient() ##creates client##
client.headers = newHttpHeaders({ "Content-Type": "application/json" })
let body = %*{
"data": "Test" ##sets up body message##
}
let response = client.request(input, httpMethod = HttpPost, body = $body)
request() ##sends request
echo response.status ##outputs status##
if response.status == 200: 
proc createFile(dir: string){
request.txt 
}
##If the status is 200 it creates a file called request.txt and start the other functions##
  output() #Outputs IP.
  resent() #Resends Data
  switch() #Changes IP to original
  Log() #Logs traffic
else:
  echo "Sorry there's an error."

File: Python/DrMITM.py
import requests
import time
import socket
import MTG
url = input("Enter URL: ")
print("sending request")
payload = {"Testing"}
r = requests.post(url, data=payload)
print(r.text)
time.sleep(1)
print("IP: "  socket.gethostname(url))
print("Note: the traffic request is stored in request.txt")
ip = print("Switching IP")
ip = url
print("Sending message to the Server")
resend()
